source/aks/upgrading-crime-aks-clusters.html.md.erb

```diff
@@ -0,0 +1,373 @@
+---
+title: Upgrading Crime AKS Clusters
+last_reviewed_on: 2025-07-24
+review_in: 12 months
+weight: 30
+---
+
+# Upgrading Crime AKS Clusters
+
+# Node image only upgrade
```

Comment:
Should be a `##` header to maintain formatting.

All following should be updated with an additional `#` to maintain formatting.